source/includes/list-table-shell-properties.rst

```diff
@@ -0,0 +1,82 @@
+.. list-table::
```

Comment:
[Note for review]

This table was already present in the copy. I've just moved it out to an include so it can be re-used across multiple settings pages. Feel free to ignore this table for this review.